src/app/meds-n-labels/activity-1/activity-1.component.scss

```diff
@@ -5,32 +5,46 @@
 }
 
 .medicine {
-    height: 100px;
-    width: 100px;
+    height: 60px;
+    width: 60px;
 }
 
 .des {
-    height: 135px;
+    height: 80px;
+    width: 400px;
+    margin-bottom: 60px;
     overflow: hidden;
+    font-size: $font-size-small;
+    color: $white;
+    background-color: $light-blue;
 }
 
 .heading-info {
     color: $orange;
 }
 
-@include match-elements(0, 30%, 60%);
+@include match-elements(33%, 40%, 55%);
```

Comment:
can these values be stored as variables ?

Reply:
Done.